packages/rpc-core/src/rpc-methods/getTokenAccountBalance.ts

```diff
@@ -0,0 +1,27 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+
+import { Commitment, RpcResponse } from './common';
+
+type GetTokenAccountBalanceApiResponse = RpcResponse<{
+    /** The raw balance without decimals, a string representation of u64 */
+    amount: string;
+    /** Number of base 10 digits to the right of the decimal place */
+    decimals: number;
+    /** @deprecated The balance, using mint-prescribed decimals */
+    uiAmount: number | null;
+    /** The balance as a string, using mint-prescribed decimals */
+    uiAmountString: string;
+}>;
```

Comment:
Can you find every hand-hewn type like this and replace it with this?

https://github.com/solana-labs/solana-web3.js/blob/958cc6b5ceaa8eddc79d46ed133e2ceac4593476/packages/rpc-core/src/rpc-methods/common.ts#L98-L103

Reply:
Also do a quick once-over looking for `: string` where we should be using `: StringifiedBigInt`.